components/form/hooks/useChildren.ts

```diff
@@ -0,0 +1,13 @@
+import toArray from 'rc-util/lib/Children/toArray';
+import type { FormItemProps } from '../FormItem';
+
+export default function useChildren(
+  children?: FormItemProps['children'],
+): FormItemProps['children'] {
+  if (typeof children === 'function') {
+    return children;
+  }
+
+  const childList = toArray(children);
+  return childList.length <= 1 ? childList[0] : childList;
+}
```

Comment:
Hi, I'm trying to use form.item in version 5.6.1, but when I pass the value to children prop I get the `undefined` value in the children component. I think this useChildren is causing it. When I downgrade to antd@5.4.2 (when this useChildren wasn't added yet), it works fine.